index.js

```diff
@@ -124,6 +125,61 @@ app.post('/', function(req, response) {
 				if (error) throw new Error(error);
 				console.log(body);
 			});
+				
+			var options1 =   {
```

Comment:
Different options have different type of data i.e in one option type is location and in other its text.